packages/react-examples/src/react-tooltip/Tooltip/Tooltip.stories.tsx

```diff
@@ -53,7 +53,15 @@ export const Basic = () => {
         >
           <button>Formatted content</button>
         </Tooltip>
-        <Tooltip content="This tooltip targets the red square" targetRef={targetRef} triggerAriaAttribute="describedby">
+        <Tooltip
+          content="This tooltip targets the red square"
+          onBeforeShow={(_, data) => {
```

Comment:
Again, if this component can be controlled, then there would be no need for lifecycle methods like this. It's something that reminds me of all the custom event callbacks in v8 `Callout` that end up getting lost and eventually deprecated.